custom/ahoy.yml

```diff
@@ -11,6 +11,10 @@ commands:
       echo -e "Type \`\033[0;32mahoy my\033[0m\` for a list of custom commmands."
       echo
 
+  drush-wo-line-wrap:
```

Comment:
```suggestion
  drush-local:
```

Reply:
Just a more succinct command name! `ahoy drush-wo-line-wrap` would be cumbersome :D :shipit:

Reply:
100% 👍